habitat-lab/habitat/config/benchmark/rearrange/play_stretch.yaml

```diff
@@ -16,13 +16,16 @@ habitat:
       arm_action:
         type: "ArmAction"
         arm_controller: "ArmRelPosKinematicReducedActionStretch"
-        grip_controller: "MagicGraspAction"
+        grip_controller: "GazeGraspAction"
         arm_joint_mask: [1,0,0,0,1,1,1,1,1,1]
         arm_joint_dimensionality: 10
         grasp_thresh_dist: 0.15
         disable_grip: False
         delta_pos_limit: 0.0125
         ee_ctrl_lim: 0.015
+        gaze_distance_range: [0.1, 3.0]
+        center_cone_angle_threshold: 20.0
+        center_cone_vector: [0.0, 1.0, 0.0]
```

Comment:
Hi Jimmy! You may want to update the resolution and hfov of `head_panoptic_sensor` in this config.

Reply:
Thank you @yvsriram! Added `head_panoptic_sensor` to the config. Thank you so much for this!